docs/metrics/object-detection.md

```diff
@@ -0,0 +1,121 @@
+    [Precision-Recall (PR) curves](./pr-curve.md) and then macro-averaging those scores across all classes. mAP is a
+    comprehensive indicator of a model's performance across multiple categories.
+
+    $$
+    \begin{align}
+    \text{mAP} = \frac{1}{N} \sum_{\text{class}}^{\text{all classes}} \text{AP}_\text{class}
+    \end{align}
+    $$
+
+    !!! info "Guide: Averaging Methods"
+
+        Read the [averaging methods](./averaging-methods.md) guide if you're not familiar with "macro" and "micro"
+        terminology.
+
+## Evaluation Plots for Object Detection Models
+
+There are several common plots used to analyze the performance of object detection models.
+
+??? info "Advanced Plot Insights"
+
+    Suppose we are doing `person object detection` like within the first image above. It would be interesting to see
+    how performance differs by different test cases or different characteristics of data. With different lines on
+    plots representing different groups, such as `body part`, `race`, `bounding box size`, `image brightness`, etc.
+    plots easily explain where performance drops.
+
+    <center>
+    ![pr_by_testcase.png](../assets/images/metrics-tasks-od-prcurve-light.png#only-light)
+    ![pr_by_testcase.png](../assets/images/metrics-tasks-od-prcurve-dark.png#only-dark)
+    </center>
+
+    From the plot above, we see how a particular model suffers in performance when only a person's `arm` is shown.
+
+1. [**Precision Recall (PR) Curves**](./pr-curve.md): The PR curve plots precision and recall within a unit square.
+    The greater the area under the curve, the more performant your model is. Typically, there is a tradeoff between
+    precision and recall, which you can read in detail [here](./pr-curve.md).
+
+    <center>
+    ![pr.png](../assets/images/metrics-prcurve-light.png#only-light)
+    ![pr.png](../assets/images/metrics-prcurve-dark.png#only-dark)
+    </center>
+
+2. [**Confusion Matrix**](./confusion-matrix.md): Particularly for multiclass object detection, a confusion matrix
+    displays the actual classes against the predicted classes in a table. This allows for a clear understanding of
+    the model's performance with respect to classifying classes.
+
+    <center>
+    ![pr.png](../assets/images/metrics-confusion-matrix-light.png#only-light)
+    ![pr.png](../assets/images/metrics-confusion-matrix-dark.png#only-dark)
+    </center>
+
+    From the plot above, we can see if the model is confused when detecting for cats and dogs.
+
+Note that any plot suitable for a classification task is also suitable for an object detection task, as object
+detection is a combination of a classification and localization task. However, it is important to consider what
+matters to your project and design custom plots to describe your model's performance in the best way possible.
+For example: an F1-Score vs confidence threshold plot, a histogram of IoUs, or mAP across different IoUs.
```

Comment:
`mAP` isn't a plot.   Did you mean a PR curve?

Reply:
I meant a plot like [this](https://www.researchgate.net/figure/Measured-mAP-for-different-IoU-threshold-values-for-all-considered-fusion-approaches_fig4_358237740) written as `mAP across different IoUs`